### PR TITLE
Refined some tests and added some documentation. Part of user story 3 of sprint 8.

### PR DESCRIPTION
**Pull request by MatthijsJWH**
Tests for selling land have now been added to all necessary test modules. Also added comments to the test modules to create some documentation on the structure and purpose of the different tests.

The test can unfortunately not be run through travis, but all tests pass localy.

### CODE REVIEW

**Comment by lisv** [comment]
I approve. All tests pass and the code looks fine.

File: lib/tygron [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 41169d27e37352fc367d4bdfbfa9cf7e3480a1f6
```
Comment (JoshCode) [inline lib/tygron]: This is the incorrect submodule commit reference

Comment (MatthijsJWH) [inline lib/tygron]: I have not edited this one myself, can you maybe correct it?

**Comment by JoshCode** [comment]
Submodule was fixed, waiting for the builds and then it can be merged.

**Comment by MHMCD** [comment]
Looks good and all the tests work. I approve

**Comment by shdshdw** [comment]
Approved these tests are improved and it does not break anything.

**Pull request merged**